http/get_simple/python/client/client.py

```diff
@@ -35,13 +39,12 @@
       pass
 
 # or:
-#with pa.ipc.open_stream(buffer) as reader:
+#with pa.ipc.open_stream(response) as reader:
 #  schema = reader.schema
 #  batches = [b for b in reader]
 
 end_time = time.time()
 execution_time = end_time - start_time
 
-print(f"{len(buffer)} bytes received")
```

Comment:
Could perhaps use `response.tell()` if we still want to print out this information

Reply:
I get `io.UnsupportedOperation: seek` when I call `response.tell()`

Reply:
Ah, too bad.